database/migrations/2024_05_03_122907_create_areas_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('areas', function (Blueprint $table) {
+            $table->id();
+            $table->string('name', 50);
+            $table->unsignedBigInteger('attribute_id');
+            $table->unsignedBigInteger('fee_id');
+            $table->string('address', 50);
```

Comment:
adjust to 255

Reply:
Thank you for your suggestion. I will change to 255.
$table->string('address', 255);